components/RecipeForm.js

```diff
@@ -0,0 +1,21 @@
+import StyledInput from './StyledComponents/StyledInput';
+import StyledButton from './StyledComponents/StyledButton';
+import StyledForm from './StyledComponents/StyledForm';
+
+export default function RecipeForm() {
+  return (
+    <>
+      <StyledForm>
+        <label>
+          <StyledInput
+            type="text"
+            size="30"
+            placeholder="Add recipe title"
+          ></StyledInput>
```

Comment:
```suggestion
          />
```

use a self-closing tag